### PR TITLE
Feat/que pg

### PR DESCRIPTION
**Pull request by aurobindoGupta**
image
image
image

### CODE REVIEW

File: components/quePg/styles.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import url(/styles.css);
+
+:root{
+  line-height: 2rem;
+ 
+}
+.page-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (aniketsingh13) [inline components/quePg/styles.css:9]: can be done using classes

File: components/quePg/styles.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  width: 100%;
+  height: 100%;
+  max-height: fit-content;
+  justify-content: space-evenly;
+}
+.quizz-pg{
+  display: flex;
+  flex-direction: column;
```
Comment (aniketsingh13) [inline components/quePg/styles.css:17]: here also

**Review by aniketsingh13 — APPROVED** [review]
remove extra space in your code other thing look good to me

File: components/quePg/styles.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+.ans:hover {
+  transform: scale(0.98);
+}
+.rightAns {
+  background: green;
+  color: var(--background-color);
+}
+.wrongAns {
+  background: rgb(226, 30, 30);
```
Comment (surbhi84) [inline components/quePg/styles.css:71]: hex codes can be used for color

File: components/quePg/wrongAns.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Aurobindo Gupta" />
+  <meta name="description" content="Personal Quizz Website" />
```
Comment (surbhi84) [inline components/quePg/wrongAns.html:9]: you can write harry potter quiz website instead of personal.

**Review by surbhi84 — APPROVED** [review]
Left some comments,rest looks good to me.

File: components/quePg/styles.css (hunk 3 above)
Comment (aurobindoGupta) [reply components/quePg/styles.css:71]: changed in diff branch

**Pull request merged**